File: Q2_2.py
#vectors
A=[1,2,3]
B=[4,5,6]
c=[]
# finding A+B
i=0
x=0
while i<len(A) and i<len(B):
    x=A[i]+B[i]
    c.append(x)
    i=i+1
print("sum: ",c)

#find A.B
s=0
j=0;
while j<len(A) and j<len(B):
    d=A[j]*B[j]
    s=s+d
    j=j+1
print("dot product:",s)
#output: sum:[5 7 9]
#       dot product:32

File: Q2_2.c
//vectors A+B and A.B

#include<stdio.h>
#include<conio.h>

int main(){
    int A[3]={1,2,3};
    int B[3]={4,5,6};
    int C[3];
    int d; int s=0;
    for(int i=0;i<3;i++){
        C[i]=A[i]+B[i];
    }
    
    //sum

    printf("Sum is:");
    for(int j=0;j<3;j++){
        printf("%d ",C[j]);
    }
    printf("\n");
    
    //dot product

    for(int k=0;k<3;k++){
        d=A[k]*B[k];
        s=s+d;
    }
    printf("A.B is: %d",s);
    return 0;
    //Output: Sum is: 5 7 9
    //        A.B is:32

}